android/app-newm/src/main/java/io/newm/navigation/Navigation.kt

```diff
@@ -17,20 +17,19 @@ import io.newm.screens.library.LibraryScreen
 import io.newm.screens.library.NFTLibraryScreen
 import io.newm.screens.profile.edit.ProfileRoute
 import io.newm.screens.profile.view.ProfileViewScreen
-import io.newm.shared.models.Song
+import io.newm.shared.repositories.NFTTrack
```

Comment:
repositories shouldn't be exposed to the Presentation layer right?  The whole data layer (API/DAO) should be internal to the KMM module.

Reply:
That is correct, all classes in KMM will either be internal or private except for use cases and data models.